src/services/jsDoc.ts

```diff
@@ -1,48 +1,92 @@
 /* @internal */
 namespace ts.JsDoc {
     const jsDocTagNames = [
+        "abstract",
+        "access",
+        "alias",
+        "arg",
+        "argument",
+        "async",
         "augments",
         "author",
-        "argument",
         "borrows",
         "callback",
         "class",
+        "classDesc",
+        "const",
         "constant",
         "constructor",
         "constructs",
+        "copyright",
         "default",
         "deprecated",
         "description",
+        "emits",
+        "enum",
         "event",
         "example",
+        "exports",
         "extends",
+        "external",
         "field",
+        "file",
         "fileOverview",
+        "fires",
+        "func",
         "function",
+        "generator",

```

Comment:
Is this one really useful? Even the JSDoc website says the scenario is pretty niche.

> In general, you do not need to use this tag, because JSDoc automatically detects asynchronous functions and identifies them in the generated documentation. However, if you are writing a virtual comment for an asynchronous function that does not appear in your code, you can use this tag to tell JSDoc that the function is asynchronous.